src/preprocess_data.py

```diff
@@ -0,0 +1,117 @@
+def pre_process(path):
```

Comment:
You could probably have a tester for the functions in this script.

Reply:
Will do when I complete the functionalities! Right now I have only completed data loading (to S3), create db (SQL and RDS)!..